### PR TITLE
add nomic-embed-vision-v1.5

### PR DESCRIPTION
**Pull request by Dylancouzon**
Do I fully know what I'm doing? Not really
Was AI involved? Absolutely
Did I do my best testing and verifying the contents? Yes sir

# new: add nomic-embed-vision-v1.5

Adds `nomic-ai/nomic-embed-vision-v1.5` and its quantized `-Q` variant to `ImageEmbedding`.

Both models use the same 768-dimensional embedding space as the existing `nomic-embed-text-v1.5`, enabling text-to-image and image-to-text search without extra alignment.

## How

`nomic-embed-vision-v1.5` returns `last_hidden_state` with shape `[batch, 197, 768]` instead of a pooled embedding. This PR adds a `NomicVisionEmbedding` subclass that takes the CLS token at index `0` and L2-normalizes it, matching the model's reference recipe.

This mirrors the existing text-side pooling override in `PooledEmbedding`. The base class and other image models are unchanged.

## Canonical values

For fp32, canonical values were computed from the original HF Transformers model and match the ONNX path bit-for-bit (`max abs diff 0.0`):

```python
from transformers import AutoImageProcessor, AutoModel
import torch, torch.nn.functional as F
from PIL import Image

proc = AutoImageProcessor.from_pretrained("nomic-ai/nomic-embed-vision-v1.5")
model = AutoModel.from_pretrained("nomic-ai/nomic-embed-vision-v1.5", trust_remote_code=True).eval()
inp = proc(Image.open("tests/misc/image.jpeg"), return_tensors="pt")
with torch.no_grad():
    emb = F.normalize(model(**inp).last_hidden_state[:, 0], p=2, dim=1)
print(emb[0, :10])  # canonical vector for nomic-embed-vision-v1.5
```

For `-Q`, there is no torch reference for the quantized graph, so the canonical value is the quantized ONNX output on `linux/amd64` with cosine `0.995` to fp32. The test is skipped on macOS because ARM int8 kernels diverge, matching the existing handling for `nomic-embed-text-v1.5-Q`.

Models added:

- `nomic-embed-vision-v1.5`: 768 dims, 0.37 GB, Apache-2.0
- `nomic-embed-vision-v1.5-Q`: 768 dims, 0.1 GB, Apache-2.0

---

### Checklist:

- [x] Have you followed the guidelines in our Contributing document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### New Feature Submissions:

- [x] Does your submission pass the existing tests?
- [x] Have you added tests for your feature?
- [x] Have you installed `pre-commit` with `pip3 install pre-commit` and set up hooks with `pre-commit install`?

### New models submission:

- [x] Have you added an explanation of why it's important to include this model?
- [x] Have you added tests for the new model? Were canonical values for tests computed via the original model?
- [x] Have you added the code snippet for how canonical values were computed?
- [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: fastembed/image/onnx_embedding.py [hunk 1]
```diff
@@ -215,3 +235,42 @@ def init_embedding(self, model_name: str, cache_dir: str, **kwargs: Any) -> Onnx
...
+    def _post_process_onnx_output(
+        self, output: OnnxOutputContext, **kwargs: Any
+    ) -> Iterable[NumpyArray]:
+        # nomic-embed-vision emits last_hidden_state, which onnx_embed flattens to
+        # (batch, tokens * dim). Recover the token axis, take the CLS token (index 0) and normalize,
+        # matching the reference F.normalize(last_hidden_state[:, 0], p=2, dim=1).
+        dim = self.model_description.dim
+        assert dim is not None, "Model description is missing the embedding dim"
+        hidden_states = output.model_output.reshape(output.model_output.shape[0], -1, dim)
+        return normalize(hidden_states[:, 0])
```
Comment (joein) [inline fastembed/image/onnx_embedding.py]: This should be in a separate file (please take a look how it is implemented with text dense models)

File: fastembed/image/onnx_embedding.py [hunk 2]
```diff
@@ -215,3 +235,42 @@ def init_embedding(self, model_name: str, cache_dir: str, **kwargs: Any) -> Onnx
             threads=1,
             **kwargs,
         )
+
+
+class NomicVisionEmbedding(OnnxImageEmbedding):
```
Comment (joein) [inline fastembed/image/onnx_embedding.py]: I think this should be called differently

In text dense models instead of calling a class by the first model  family which requires a certain way of postprocessing, we call it by the postprocessing type this class is implementing. E.g. when we need to pool and normalize - we call it PooledNormalizedEmbedding

If pooling is not required, then we can call it just NormalizedEmbedding

**Review by joein — CHANGES_REQUESTED** [review]
Hey @Dylancouzon , thanks for the PR

There are some minor things I think we need to fix before merging

File: fastembed/image/onnx_embedding.py (hunk 1 above)
Comment (Dylancouzon) [reply fastembed/image/onnx_embedding.py]: Done

File: fastembed/image/onnx_embedding.py (hunk 2 above)
Comment (Dylancouzon) [reply fastembed/image/onnx_embedding.py]: Renamed to NormalizedEmbedding

**Comment by Dylancouzon** [comment]
Changes applies, all tests pass green

**Review by joein — APPROVED** [review]
